pentagon/cli.py

```diff
@@ -4,12 +4,21 @@
 import logging
 import traceback
 import pentagon
+import yaml
+import json
+
+from pydoc import locate
```

Comment:
Does pydoc need to be added to the dependency list? I can't seem to find it listed anywhere to be installed.

Reply:
pydoc is native

Reply:
Ah nice, I haven't used it as a lib ever.